### PR TITLE
Module renaming

### PR DESCRIPTION
**Pull request by KennethSkylight**
Ticket: https://reportstream-service-request.clickup.com/t/3883nkh

Description

As we move our files to our new SDK repo, we wanted to modify the folder names to reflect the new state of the project. We have changed phdi_building_blocks to just phdi.

### CODE REVIEW

File: examples/ECR Data Mart Creation.ipynb [hunk 1]
```diff
@@ -210,9 +210,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "myenv",
+   "display_name": "Python 3.9.13 64-bit",
```
Comment (spencer-skylight) [inline examples/ECR Data Mart Creation.ipynb:213]: I think you can get rid of these lines that cause some thrashing between machines in source control by running:
```
pip install nb-clean
nb-clean clean "examples/ECR Data Mart Creation.ipynb"
```

This is good fodder for the doc changes I'm working on too - I'll try to add this for posterity.

**Review by spencer-skylight — CHANGES_REQUESTED** [review]
Just one minor suggestion for the notebook change.

We'll rearrange a little more once we move over to phdi-sdk - possibly move this to the base project level in that repo.  The module renaming is a good next step though.

File: examples/ECR Data Mart Creation.ipynb (hunk 1 above)
Comment (KennethSkylight) [reply examples/ECR Data Mart Creation.ipynb:213]: Done

**Comment by spencer-skylight** [comment]
This PR has been ported to https://github.com/CDCgov/phdi-sdk

**Pull request closed without merging**